packages/ui-components/src/Tabs/Tabs.js

```diff
@@ -80,6 +80,14 @@ export default class Tabs extends PureComponent {
       this.setState(newState, this.onTabsUpdated)
     }
     this.props.onSelectTab(selected)
+
+    let tabIndex
+    this.allTabs.forEach( (item, index) => {
+      if( item.key === tab.key) {
+        tabIndex = index
+      }
+    })
+    tabIndex || tabIndex === 0 ? this.header.current.moveCurrentTab(tabIndex) : ''
```

Comment:
this line confused me, if the condition result is false, how javascript will execute it